### ISSUE TITLE
test_to_records fails on BE boxes: (0L, 'a') != (0L, 'a')

### ISSUE THREAD

**Issue by yarikoptic** [pydata/pandas#8996]
see e.g. https://buildd.debian.org/status/fetch.php?pkg=pandas&arch=sparc&ver=0.15.1%2Bgit125-ge463818-1&stamp=1417703234&file=log
```
======================================================================
FAIL: test_to_records (pandas.tests.test_categorical.TestCategoricalAsBlock)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/«BUILDDIR»/pandas-0.15.1+git125-ge463818/debian/tmp/usr/lib/python2.7/dist-packages/pandas/tests/test_categorical.py", line 2393, in test_to_records
    tm.assert_almost_equal(result,expected)
  File "das/src/testing.pyx", line 58, in pandas._testing.assert_almost_equal (pandas/src/testing.c:2745)
  File "das/src/testing.pyx", line 93, in pandas._testing.assert_almost_equal (pandas/src/testing.c:1830)
  File "das/src/testing.pyx", line 143, in pandas._testing.assert_almost_equal (pandas/src/testing.c:2618)
AssertionError: (0L, 'a') != (0L, 'a')
```

**Comment by jreback** [comment]
dupe of #10425 (and closed by #10438)

**Comment by jreback** [comment]
@yarikoptic the fix was merged in #10438 should fix the big-endian issues. If there are most pls post.

**Comment by yarikoptic** [comment]
On Fri, 26 Jun 2015, jreback wrote:

>    @yarikoptic the fix was merged in #10438 should fix the big-endian issues.
>    If there are most pls post.

ok... I will then attempt a fresh build across

Cheers!
-- 
Yaroslav O. Halchenko, Ph.D.
http://neuro.debian.net http://www.pymvpa.org http://www.fail2ban.org
Research Scientist,            Psychological and Brain Sciences Dept.
Dartmouth College, 419 Moore Hall, Hinman Box 6207, Hanover, NH 03755
Phone: +1 (603) 646-9834                       Fax: +1 (603) 646-1419
WWW:   http://www.linkedin.com/in/yarik
